File: Technology Fundamentals/02.Exercises - Intro and Basic Syntax/CSharp/More Exercise/02.EnglishNameLastDigit/EnglishNameLastDigit.cs
using System;

namespace _02.EnglishNameLastDigit
{
    public class EnglishNameLastDigit
    {
        public static void Main()
        {
            int number = int.Parse(Console.ReadLine());
            int lastDigit = number % 10;

            string result = string.Empty;
            switch (lastDigit)
            {
                case 0:
                    result = "zero";
                    break;
                case 1:
                    result = "one";
                    break;
                case 2:
                    result = "two";
                    break;
                case 3:
                    result = "three";
                    break;
                case 4:
                    result = "four";
                    break;
                case 5:
                    result = "five";
                    break;
                case 6:
                    result = "six";
                    break;
                case 7:
                    result = "seven";
                    break;
                case 8:
                    result = "eight";
                    break;
                case 9:
                    result = "nine";
                    break;
            }
            Console.WriteLine(result);
        }
    }
}

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/PHP/02.More Exercises/EnglishNameLastDigit.php
<?php
$number = intval(readline());

$lastDigit = $number % 10;

switch ($lastDigit) {
    case 0:
        echo "zero";
        break;
    case 1:
        echo "two";
        break;
    case 2:
        echo "two";
        break;
    case 3:
        echo "three";
        break;
    case 4:
        echo "four";
        break;
    case 5:
        echo "five";
        break;
    case 6:
        echo "six";
        break;
    case 7:
        echo "seven";
        break;
    case 8:
        echo "eight";
        break;
    case 9:
        echo "nine";
        break;
}
?>